src/views/Home.vue

```diff
@@ -222,6 +222,12 @@ export default {
               .filter((module) => module),
           }));
       }
+
+      const cachedPlan = localStorage.getItem('plan');
+      if (cachedPlan != null) {
```

Comment:
could be a ternary return expression

Reply:
how? I don’t return within the if

Reply:
would work with return cachedPlan != null ? : []

or even

return (window.location.hash = cachedPlan) ?? []

as the return value would correctly be the value of the plan anyways, no? I'm fine with it if you don't care enough but it seems so hacky

Reply:
It didn't work with the second option. No idea why :/